tutorials/energy/query-energy.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Energy consumption data tutorial - query the data
+excerpt: Energy consumption data
+products: [cloud, mst, self_hosted]
+keywords: [tutorials, query]
+tags: [tutorials, beginner]
+layout_components: [next_prev_large]
+content_group: Analyze energy consumption data
+---
+
+# Query the data
+
+When you have your dataset loaded, you can start constructing some queries to
+discover what your data tells you. In this section, you learn how to construct
+queries, to answer these questions:
```

Comment:
I have a line about Toolkit that I used in the blockchain one here.

Reply:
done!